vllm/core/block/block_table.py

```diff
@@ -89,7 +91,8 @@ def allocate(self,
 
     def append_token_ids(self,
                          token_ids: List[int],
-                         num_lookahead_slots: int = 0) -> None:
+                         num_lookahead_slots: int = 0,
+                         num_computed_slots: Optional[int] = None) -> None:
```

Comment:
Can you explain why we need this? is it sliding-window-specific or sliding-window+chunked-prefill specific?